src/model/interfaces/order.ts

```diff
@@ -17,4 +18,5 @@ export interface Order {
     billingAddress?: Address
     paymentInformation?: PaymentInformation
     calculateTotalCost(): number
+    vatNumber: string
```

Comment:
Personally, I would prefer this to be optional. Also, I would have placed it above the `calculateTotalCost` method to have properties and methods grouped together.

Reply:
But maybe there is a good reason why it should not be optional. Since there is no clear right and wrong, I will not request any changes. This is up for discussion and you as the author of the changes should decide.

Reply:
I can make it optional, but then the order and invoice service will also be affected, next time it would be good to mention this early on.